content/en/continuous_integration/intelligent_test_runner/how_it_works.md

```diff
@@ -0,0 +1,40 @@
+---
+title: How It Works
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/"
+    tag: "Blog"
+    text: "Streamline CI testing with Datadog Intelligent Test Runner"
+  - link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+    tag: "Blog"
+    text: "Monitor all your CI pipelines with Datadog"
+---
+
+## What is Intelligent Test Runner?
+
+Intelligent Test Runner is Datadog's test impact analysis solution. Test impact analysis is a technique that has been getting popular in the past couple of decades, but is usually hard and time-consuming to implement. Intelligent Test Runner cuts through that complexity with a simple setup process.
+
+Test impact analysis maps each test to the set of code files in your repository that your test exercises, with the goal of reducing the time taken by the testing phase of your CI. This is achieved by analyzing the code changes you are trying to test, and skipping the tests that do not impact your code changes.
```

Comment:
> Test impact analysis maps each test to the set of code files in your repository that your test exercises, with the goal of reducing the time taken by the testing phase of your CI.

It feels like the phrase suggests that mapping tests to sets of files is going to reduce testing time somehow. I'd  suggest elaborating a bit, something along these lines:

"... with the goal of skipping the tests that are not affected by the code changes that are being introduced, in order to reduce the time taken by the testing phase of your CI"